docs/admin/admin-ui/access-control.md

```
@@ -0,0 +1,230 @@
+    Rel(Frontend, Cedarling, "Check user permissions<br>for UI features and actions", "TBAC Authorization")
+```
+
+## Policies
+
+The Cedar policies in the Policy Store empowers the administrator with following:
+
+### MAnage the access control in Admin UI
+
+The Cedar policies are rules which decides if the logged in user can perform Read, Write or Delete action on a feature or not. By writing the appropriate cedar policies the administrator can manage access control in Admin UI. For e.g the below policy allows the user with role **admin** to perform **Read**, **Write** or **Delete** actions on all the features under the parent group **AuthServerAndConfiguration**.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)